specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2023-10-01-preview/account.json

```diff
@@ -0,0 +1,2681 @@
+            "$ref": "./examples/Collections_CreateOrUpdateCollection.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "Deletes a Collection entity.",
+        "operationId": "Collections_DeleteCollection",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "collectionName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Collections_DeleteCollection": {
+            "$ref": "./examples/Collections_DeleteCollection.json"
+          }
+        }
+      }
+    },
+    "/collections": {
+      "get": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "List the collections in the account.",
+        "operationId": "Collections_ListCollections",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CollectionList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the provider.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Collections_ListCollections": {
+            "$ref": "./examples/Collections_ListCollections.json"
+          }
+        }
+      }
+    },
+    "/collections/{collectionName}/getChildCollectionNames": {
+      "get": {
+        "tags": [
+          "Collection"
+        ],
+        "description": "Lists the child collections names in the collection.",
+        "operationId": "Collections_ListChildCollectionNames",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "collectionName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "query",
+            "name": "$skipToken",
```

Comment:
Azure services usually don't use skip tokens but let the server drive the pagination through the opaque nextlink. If you need the client to drive the pagination you would usually support skip/top. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections and https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#pagination